.github/workflows/hide_bot_comments.yml

```diff
@@ -0,0 +1,24 @@
+name: Hide Bot Comments
+on:
+  push:
+    branches: [master]
+  pull_request:
+    types: [opened, synchronize, reopened]
+    branches: [master]
+
+jobs:
+  hide-bot-comments:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v1
+        with:
+          python-version: "3.8"
+      - name: Install Requests
+        run: |
+          pip install requests
+      - name: Hide Bot Comments
+        run: |
+          python .github/workflows/main.py
```

Comment:
It is probably better to name this more concrete, given that this isn't the only GHA workflow

Reply:
It also does not follow the normal github action structure. Not sure that relevant here but just an FYI

Reply:
I am not sure what exactly doesn't follow? I mostly copy-pasted from https://github.com/bincrafters/community/blob/287e90d6371bae02586c2bc463ab32ef0fa95e89/.github/workflows/auto-merge.yml#L29 and https://github.com/conan-io/conan-center-index/pull/4876/files. it would be nice if you could be more specific on what should be changed.

Reply:
I apologize! I answered late last night and that was not helpful.

Typically the `.github/` folder looks like this

```
actions/ <-- suggest moving it here
    one/
    two/
workflows/
    alpha.yml
    beta.yml
bot.yml
```

Very often I see the script just in the "actions" directory but you could wrap it in a local action https://docs.github.com/en/actions/creating-actions/metadata-syntax-for-github-actions#runs-for-composite-run-steps-actions